src/amo/api/addonInfo.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+import invariant from 'invariant';
+
+import { callApi } from 'core/api';
+import type { ApiState } from 'core/reducers/api';
+
+export type GetAddonInfoParams = {|
+  api: ApiState,
+  slug: string,
+|};
+
+export type ExternalAddonInfoType = {|
+  eula: string | null,
+  privacyPolicy: string | null,
```

Comment:
I did this to avoid having a variable with an underscore in it in other modules, as eslint was complaining about that, but I realize now that there are other ways around that. Undoing this will address most of your other comments too. :-)